src/main/java/com/skynest/tests/tests/RegistrationTest.java

```diff
@@ -1,33 +1,43 @@
 package com.skynest.tests.tests;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
 import com.skynest.tests.files.EndpointConstants;
-import com.skynest.tests.files.Payload;
-import com.skynest.tests.files.ReusableMethods;
-import io.restassured.path.json.JsonPath;
+import com.skynest.tests.files.RegistrationPayload;
+import com.skynest.tests.files.RegistrationResponse;
+import com.skynest.tests.utils.BaseTransformer;
+import io.restassured.response.Response;
 import org.testng.Assert;
 import org.testng.annotations.DataProvider;
 import org.testng.annotations.Test;
-import static io.restassured.RestAssured.*;
 
-public class RegistrationTest extends BaseTest{
+import static io.restassured.RestAssured.given;
+
+public class RegistrationTest extends BaseTest {
     @Test(dataProvider = "UserData")
-    void registering_new_valid_user_should_return_specified_response(String email, String pass, String name, String surname, String phoneNo, String address){
-        String response = given().log().all().header("Content-Type","application/json")
-                .body(Payload.userRegistrationData(email,pass,name,surname,phoneNo,address))
+    void registering_new_valid_user_should_return_specified_response(String email, String password, String name, String surname, String phoneNumber, String address) throws JsonProcessingException {
+
+        RegistrationPayload registrationPayload =new RegistrationPayload(email,password,name,surname,phoneNumber,address);
+
+        Response response = given().log().all().header("Content-Type", "application/json")
```

Comment:
You could use already available enums for the header key/value pairs, such as these 
```
import static org.apache.http.HttpHeaders.CONTENT_TYPE;
import static org.apache.http.entity.ContentType.APPLICATION_JSON;
```